keyboards/melgeek/z70ultra/z70ultra.c

```diff
@@ -0,0 +1,165 @@
+#include "z70ultra.h"
+#include "config.h"
+#include "is31fl3741.h"
+
+
+const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
+    {0, 28, 1, 30, 1, 29, 1}, /* RGB10 */
```

Comment:
Not really a fan of this, would it be possible to change the 3741 driver to be more like the others here? The registers on the other page could perhaps be distinguished by adding 0x100.
eg.
![image](https://user-images.githubusercontent.com/4781841/89061850-cb7aa380-d3a8-11ea-8b1c-45a0bbc9725c.png)

Reply:
please check it again.
https://github.com/qmk/qmk_firmware/pull/9202/commits/99a9610996dccba810ee36bbbffab50e8250ba6d